okta/help.md

```diff
@@ -27,6 +27,49 @@ The connection configuration accepts the following parameters:
 
 ### Actions
 
+#### Push MFA Challenge
+
+This action pushes a MFA challenge to a user's device and waits for a success or rejection.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|factor_id|string|None|True|The factor id of the user you want to push a verify to|None|
+|user_id|string|None|True|The user id of the user you want to push a verify to|None|
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|factor_status|string|False|The factor status returned by Okta for a user|
+
+Example Output:
+
+```
+```
+
+#### Get an Okta user's factors
+
+This action returns an object containing all of a user's factors for MFA.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|user_id|string|None|True|The user id of the user you want the factors of|None|
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|factors|object|False|An object containing all the factors of a user for MFA|
+
+Example Output:
+
+```
```

Comment:
Add example output from a successful run from the command-line, 2 space indented.

Reply:
👍

Reply:
Note, that we're working on automating filling out the example output so this will won't be a manual task in the future.